htmresearch/regions/ApicalTMRegion.py

```diff
@@ -341,6 +351,7 @@ def initialize(self):
         "apicalInputSize": self.apicalInputWidth,
         "cellsPerColumn": self.cellsPerColumn,
         "activationThreshold": self.activationThreshold,
+        "reducedThresholdBasal": self.reducedThresholdBasal,
```

Comment:
We'll probably want to move this line down to the `self.implementation == "ApicalTiebreak"` case.

i.e.

~~~python
params["reducedThresholdBasal"] = self.reducedThresholdBasal
self._tm = ApicalTiebreakTemporalMemory(**params)
~~~